packages/data/src/plugins/persistence/index.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { merge, isPlainObject } from 'lodash';
+import { merge, isPlainObject, reduce } from 'lodash';
```

Comment:
wasn't there a push to use native methods when available instead of lodash?

Reply:
I'll see if I can remove it 👍

Reply:
Done in [51e05d4](https://github.com/WordPress/gutenberg/pull/39449/commits/51e05d4eec1db9fa6fda2cc1da94702daeff54c8) 😄